modules/vpc-endpoints/README.md

```diff
@@ -0,0 +1,101 @@
+# AWS VPC Endpoints Terraform sub-module
+
+Terraform sub-module which creates VPC endpoint resources on AWS.
+
+The following resources are supported:
+
+- [aws_vpc_endpoint](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/vpc_endpoint)
+
+## Usage
+
+See [`examples`](./examples) directory for working examples to reference:
+
+```hcl
+module "endpoints" {
+  source = "terraform-aws-modules/vpc/aws//vpc-endpoints"
```

Comment:
```suggestion
  source = "terraform-aws-modules/vpc/aws//modules/vpc-endpoints"
```

Reply:
good catch - updated in https://github.com/terraform-aws-modules/terraform-aws-vpc/pull/635/commits/5c6a2313e49d8f537b8d0e4f45f827c62d742541